zlp-scheduler/app/controllers/concerns/scheduler_2.rb

```diff
@@ -26,14 +26,14 @@ def self.sigmoid_cost_fuc(max_cost, alpha, beta, x)
         max_cost = 100
         prior_alpha = -0.015
         prior_beta = 0.025
-        later_alpha = 0.01
-        later_beta = 0.07
+        later_alpha = 0.015
+        later_beta = 0.06
         # https://www.desmos.com/calculator
         
         if current_time < start_of_day
-            cost = max_cost - 1
-            # x = (current_time.to_f - start_of_day.to_f)/900
-            # cost = self.sigmoid_cost_fuc(max_cost, prior_alpha, prior_beta, x)
+            # cost = max_cost - 1
+            x = (current_time.to_f - start_of_day.to_f)/900
+            cost = self.sigmoid_cost_fuc(max_cost, prior_alpha, prior_beta, x)
```

Comment:
I thought we had commented this out because the morning was going to be a hard preference. Did that change? It's fine if that's the case, I'm just curious about why this got added back in.

Reply:
Thanks for your comment. I think the update was removed in some merging process.
Now I changed back it to what we agreed.